test/API/H5_api_test.c

```diff
@@ -352,54 +399,44 @@ main(int argc, char **argv)
     if (GetTestSummary())
         TestSummary(stdout);
 
-    printf("Deleting container file(s) for tests\n\n");
-
     if (GetTestCleanup()) {
-        if (H5_api_test_destroy_container_files() < 0) {
-            fprintf(stderr, "Error cleaning up global API test info\n");
-            err_occurred = true;
+        printf("Deleting container file(s) for tests\n\n");
+
+        if (H5_api_test_destroy_container_files(H5_api_test_filenames_g, H5_api_test_num_filenames_g) < 0) {
+            TestErrPrintf("Error cleaning up testing container files\n");
             goto done;
         }
     }
 
-    if (n_tests_run_g > 0) {
-        printf("%zu/%zu (%.2f%%) API tests passed with VOL connector '%s'\n", n_tests_passed_g, n_tests_run_g,
```

Comment:
This logic has been moved into the testing framework if you pass the `-summary` option